docs/source/getting-started/registering-first-model/step1-register-model.rst

```diff
@@ -0,0 +1,75 @@
+Register a Model
+=================
+
+Throughout this tutorial we will leverage a local tracking server and model registry for simplicity.
+However, for production use cases we recommend using a 
+`remote tracking server <https://mlflow.org/docs/latest/tracking/tutorials/remote-server.html>`_.
+
+Step 0: Install Dependencies
+----------------------------
+.. code-section::
+    .. code-block:: bash
+
+        pip install --upgrade mlflow
+
+Step 1: Register a Model
+--------------------------------
+
+MLflow has lots of model flavors. In the below example, we'll leverage scikit-learn's 
+RandomForestRegressor to demonstrate the simplest way to register a model, but note that you
+can leverage any `supported model flavor <https://mlflow.org/docs/latest/models.html#built-in-model-flavors>`_.
+
```

Comment:
Can you add some content explaining what the code snippets do below? Namely, you train and log an MLflow model and then register it to the model registry in the process of logging the model. Otherwise, it can be a bit confusing.